src/app/wallet/shared/util/utils.ts

```diff
@@ -145,3 +145,21 @@ export class Duration {
   }
 
   }
+
+export class AddressHelper {
+  addressRegex: RegExp = /^[pPrRTt][a-km-zA-HJ-NP-Z1-9]{25,}$/;
```

Comment:
Does this work on both testnet and mainnet?

Reply:
yes it works on both, just tested it now